source/Politics.cpp

```diff
@@ -100,7 +100,7 @@ void Politics::Offend(const Government *gov, int eventType, int count)
 				provoked.insert(other);
 			}
 		}
-		else if(abs(weight) >= .05 && count * weight)
+		else if(abs(weight) >= .05 && (count * weight) != 0.)
```

Comment:
I think it could actually just be `... && count)`, because the first check already guarantees that `weight` is nonzero.